lib/query-builders/applicant.js

```diff
@@ -18,24 +20,43 @@ module.exports = {
   myTasks: (queryBuilder, profile) => {
     queryBuilder
       .where(isApplicant(profile))
-      .andWhere('status', returnedToApplicant.id);
+      .whereIn('status', [
+        returnedToApplicant.id,
+        recalledByApplicant.id
+      ]);
   },
 
   outstanding: (queryBuilder, profile) => {
     queryBuilder
       .where(isApplicant(profile))
-      .andWhere('status', returnedToApplicant.id);
+      .whereIn('status', [
+        returnedToApplicant.id,
+        recalledByApplicant.id
```

Comment:
Looks like `myTasks` and `outstanding` return the same results? Should we bin off `myTasks` for applicants?

Reply:
perhaps, but out of scope for this, as its there currently

Reply:
it would be useful for people who have access to lots of tasks, but only care about their ones